src/js/popup.js

```diff
@@ -153,6 +153,19 @@ function loadTemplateEditor (callback = false) {
             callback();
         }
     });
+
+    // Check the "rate" flag. If the flag is not set, display "rate it now" button
+    chrome.runtime.sendMessage({JDTIfunction: 'getRateStatus'}, function (response) {
+        if (response.status === 'success') {
+            if (response.data === true) {
```

Comment:
if the response is a boolean or null you can just check directly. ie `if (response.data) { ...`